taskipy/task_runner.py

```diff
@@ -17,11 +21,24 @@ def run(self, task_name: str, args: List[str]) -> int:
         for command in task.commands:
             if task.runner is not None:
                 command = f'{task.runner} {command}'
-
             process = subprocess.Popen(
                 command, shell=True, cwd=Path(self.__pyproject_path).parent
             )
 
+            def send_signal_to_task_process(signum: int, frame) -> None:  # pylint: disable=unused-argument
+                # pylint: disable=W0640
+                psutil_process_wrapper = psutil.Process(process.pid)
+
+                processes = psutil_process_wrapper.children(recursive=True)
+
+                innermost_process = next(
```

Comment:
This "hack" was intended for my specific SIGINT test, since in that case I wanted to simulate the user clicking CTRL+C, so I had to find the innermost process which I knew to be python. In real life, we can't know whether the task runs python, node, ruby or whatever else, so we unfortunately cannot do this ;)

Anyways see comment on line 32 - let's just iterate on the inner processes of the shell subprocess without identifying them, and send the sigterm to all of them.